CuriousKatie/Helper/Helper.swift

```diff
@@ -0,0 +1,145 @@
+        let interest25 = Interest(name: "programming", category: Category.science, gear: [Gear.highTech], environment: Environment.indoor)
+        let interest26 = Interest(name: "rock climbing", category: Category.sport, gear: [Gear.protection], environment: Environment.outdoor)
+        let interest27 = Interest(name: "running", category: Category.sport, gear: [Gear.sportClothes, Gear.runningShoes], environment: Environment.outdoor)
+        let interest28 = Interest(name: "skiing", category: Category.sport, gear: [Gear.winterClothes, Gear.skiing], environment: Environment.outdoor)
+        let interest29 = Interest(name: "skydiving", category: Category.sport, gear: [Gear.flying], environment: Environment.outdoor)
+        let interest30 = Interest(name: "snowboarding", category: Category.sport, gear: [Gear.winterClothes, Gear.snowboarding], environment: Environment.outdoor)
+        let interest31 = Interest(name: "surfing", category: Category.sport, gear: [Gear.swimwear], environment: Environment.outdoor)
+        let interest32 = Interest(name: "swimming", category: Category.sport, gear: [Gear.swimwear], environment: Environment.anywhere)
+        let interest33 = Interest(name: "traveling", category: Category.social, gear: [Gear.backpack, Gear.photo], environment: Environment.outdoor)
+        let interest34 = Interest(name: "watching movies", category: Category.social, gear: [Gear.noGear], environment: Environment.indoor)
+        let interest35 = Interest(name: "writing", category: Category.art, gear: [Gear.noGear], environment: Environment.anywhere)
+        let interest36 = Interest(name: "yoga", category: Category.sport, gear: [Gear.sportClothes], environment: Environment.anywhere)
+        
+        return [interest01,
+                interest02,
+                interest03,
+                interest04,
+                interest05,
+                interest06,
+                interest07,
+                interest08,
+                interest09,
+                interest10,
+                interest11,
+                interest12,
+                interest13,
+                interest14,
+                interest15,
+                interest16,
+                interest17,
+                interest18,
+                interest19,
+                interest20,
+                interest21,
+                interest22,
+                interest23,
+                interest24,
+                interest25,
+                interest26,
+                interest27,
+                interest28,
+                interest29,
+                interest30,
+                interest31,
+                interest32,
+                interest33,
+                interest34,
+                interest35,
+                interest36]
+    }
+    
+    /// Generate all the people
+    static func generateAllPeople() -> [Person] {
```

Comment:
This generator does't ensure the unicity of the name, does it?

Reply:
There is no function to check that the names are unique... I can check for that, although with the number of different combinations that are possible the chance of having even two identical names is probably 0.0000000000000000001 😉

Reply:
Fixed 😎